packages/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.ts

```diff
@@ -340,6 +340,12 @@ export default class MeetingInfoV2 {
       this.webex.internal.newMetrics.submitInternalEvent({
         name: 'internal.client.meetinginfo.request',
       });
+      this.webex.internal.newMetrics.submitClientEvent({
```

Comment:
what about v1 meeting info? I think we need a similar code change there too - see requestFetchInfo() in meeting-info/index.ts

Reply:
do we still use v1 meeting info? in which cases?

Reply:
If meeting is non converged then we will have v1 meeting info

Reply:
it's not about being converged, it's about the "web-meeting-wbxappapi-meetinginfo-enabled-v2" feature toggle - some customers still have it disabled and also other SDK clients that may still use `enableUnifiedMeetings=false` in sdk config.